### PR TITLE
Phalcon Vökuró frontend rewrite for Bootstrap 4.1.1 & minor other changes

### PR DESCRIPTION
**Pull request by MoHasanJ**
From couple of weeks, have been working to convert Vökuró for Bootstrap 4.1.1 in spare time. Finally its here. Came to Github specifically to share it.  Frontend is almost completely rewritten as lot has been changed from Bootstrap 2.3.1 to 4.1.1 .  Have kept most of the things to Bootstrap default so that new users can easily understand and use it as start up canvas.  

In short, major changes include

* Rewritten frontend for Bootstrap 4.1.1 and added latest related libraries.
* Open Iconic open-source icon included as recommended by Bootstrap.
* Re-factor and minor fixes of frontend code for more consistency through out including more usage of VOLT tags instead regular HTML.
* Moved design control, css classes to VOLT/VIEWS for better control of design aspect.
* Other minor frontend backend additions / improvements like removing deprecated Tag::resetInput(); and replacing with $form->clear(); where possible. 

Let me know if there are things which can be improved further.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+# Vökuró with Bootstrap 4.1.1
+
+This is sample application forked from [Phalcon Framework](https://github.com/phalcon/cphalcon) with front end (volt) rewritten for Bootstrap 4.1.1.
+
+Major changes include
```
Comment (sergeysviridenko) [inline README.md]: It must be moved to CHANGELOG, that I'll add asap.

**Comment by sergeysviridenko** [comment]
@MoHasanJ Thank you for this PR. I will add `CHANGELOG.md`, `PHP_CodeSniffer` to development branch. 
47 files are too much. Could you split this PR? Let's make the few PR to development branch when  my changes will be added. Could you?

**Comment by MoHasanJ** [comment]
@sergeysviridenko , I understand that changes are too much for single commit but as soon as   Bootstrap 4.1.1 is added, everything breaks in VIEWS , so other than fixing all, there is no other choice.  Perhaps you may give suggestion how I can split.

May be you can review and take your time. Although, from my side have done testing on all aspect for every change as I understand its lots of changes. 

And as said, I open to all suggestion, you may edit whatever you want and then apply.

**Comment by MoHasanJ** [comment]
@sergeysviridenko ,  So I fork development branch again and then apply changes gradually as you suggested, it may take again couple of days to send you full changes depending upon how much time I can spare.  However, then ofcourse can do it section wise in vokuro.

**Comment by MoHasanJ** [comment]
@sergeysviridenko I will do that however, I am still new to GIT stuff and kind of stuck at the moment. Can you explain a little more what I need to achieve when you say "Rebase your dev branch" ?'

I am starting afresh and will make changes to development branch gradually and when I am done , I will make pull request. 
OR 
If you want, can do pull request on each request so that you can review. 

Kindly advice.

**Comment by sergeysviridenko** [comment]
@MoHasanJ Do you have discord profile? We can talk there. Me `svyrydenko#5167`.

**Comment by MoHasanJ** [comment]
@sergeysviridenko  , check on Discord

**Comment by MoHasanJ** [comment]
I will make fresh pull requests as per requirement and hence closing it.

**Pull request closed without merging**